instaloader/__main__.py

```diff
@@ -95,6 +95,8 @@ def get_cookies_from_instagram(domain, browser, cookie_file='', cookie_name=''):
 
     if cookies:
         print(f"Cookies loaded successfully from {browser}")
+        print("Next: Run instaloader --login=USERNAME as it is required to download high quality media \
+            and to make full use of instaloader's features")
```

Comment:
This prints

```
Next: Run instaloader --login=USERNAME as it is required to download high quality media             and to make full use of instaloader's features
```

- There is a very big space between the words `media` and `and`
- I think it would be cool if it showed the actual username instead of `USERNAME'.

You could use [textwrap.fill](https://docs.python.org/3/library/textwrap.html#textwrap.fill) to nicely format that long message.

Reply:
Just committed the new changes, Please review.